city_scrapers/spiders/chi_ssa_27.py

```diff
@@ -0,0 +1,170 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+""" from pjsier: #we'll want to parse the agenda links from them too.
+ We also want to get the separate committee meetings and minutes as well.
+ You can get committee meetings and descriptions with div[pageareaid="2"]
+ li:last-child p and minutes with div[pageareaid="Sidebar"] li:last-child a
+"""
+
+
+class ChiSsa27Spider(CityScrapersSpider):
+    name = "chi_ssa_27"
+    agency = "Chicago Special Service Area #27 Lakeview West"
+    timezone = "America/Chicago"
+    allowed_domains = ["lakeviewchamber.com"]
+    start_urls = ["https://www.lakeviewchamber.com/ssa27"]
+
+    # main_page_title = response.css("title::text").get()
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping needs.
+        """
+        items, minutes_pdf, meeting_location, meeting = [], "", "", None
+        the_url_source = response.url
+
+        items_list = (response.css(".panel-body").getall()[4]).split("<p>")
+
+        for i in items_list:
+            items.append(re.sub("\n|\r|</p>", "", i))
+
+        for item in items:
+            if re.search(r'<div', item):
+                continue  # skip these
+
+            if re.search(r"meetings are held at", item):
+                meeting_location = self._parse_location(item)
+                continue
+
+            if re.search(r'http', item):
+                date_time, minutes_pdf = self.pdf_parse(item)
+            else:
+                date_time = self._parse_start(item)
+
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(),
+                classification=self._parse_classification(),
+                start=date_time,
+                end=self._parse_end(),  # no indication of such
+                all_day=self._parse_all_day(),  # no indication of such
+                time_notes=self._parse_time_notes(),  # haven't seen any
+                location=meeting_location,
+                links=minutes_pdf,
+                source=the_url_source,
+            )
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+        yield meeting
+
+    # We can just use "Commission" here, but if it's a committee meeting it should
+    # return the name of the committee instead.
+
+    def pdf_parse(self, item):
+        re1 = '((?:http|https)(?::\\/{2}[\\w]+)(?:[\\/|\\.]?)(?:[^\\s"]*))'  # HTTP URL 1
```

Comment:
This can likely be removed and replaced with just checking for links, and then checking if the `href` attribute ends with `.pdf`. Something like `link.attrib['href'].endswith('.pdf')`. You can also extract the `href` attribute with a CSS selector to do something like `item.css('a::attr(href)')` to get all of the `href` attributes of child links